packages/site/site-react/src/blocks/SvgImageBlock.tsx

```diff
@@ -11,7 +13,7 @@ interface SvgImageBlockProps extends PropsWithData<SvgImageBlockData> {
 }
 
 export const SvgImageBlock = withPreview(
-    ({ data: { damFile }, width = "100%", height = "auto" }: SvgImageBlockProps) => {
+    ({ data: { damFile }, width = "100%", height = "auto", ...restProps }: SvgImageBlockProps & HTMLAttributes<HTMLImageElement>) => {
```

Comment:
We should probably omit width and height from `HTMLAttributes<HTMLImageElement>` because it's defined by `SvgImageBlockProps`